src/main/scala/model/Genotype.scala

```diff
@@ -0,0 +1,42 @@
+package model
+import Alleles.AlleleKind
+import Genes.GeneKind
+
+/**
+ * Represents Phenotype of the Bunny, which is the visible traits.
+ */
+sealed trait Phenotype {
+  val visibleTraits: Map[GeneKind, AlleleKind]
+}
+
+/**
+ * Represents a standard Phenotype.
+ * @param visibleTraits traits that need to be shown
+ */
+case class StandardPhenotype(visibleTraits: Map[GeneKind, AlleleKind]) extends Phenotype
```

Comment:
Anche in questo caso mi chiedo se poi esisterà un fenotipo non standard, altrimenti proporrei di usare il companion object Phenotype e nascondere la case class per l'implementazione del trait

Reply:
Non ho capito bene come proponi di fare, però concordo che non esisterà un fenotipo non Standard, quindi se vuoi prova a modificarlo tu come dici, mi sembra un'ottima idea non avere degli "Standard" di troppo in giro.